ao_experimental/quantized_modules.py

```diff
@@ -0,0 +1,100 @@
+import torch
+from quant_primitives import dynamically_quantize_per_channel, quant_int8_dynamic_linear
+
+
+class DynamicallyQuantizedLinear(torch.nn.Module):
```

Comment:
I'd probably make sure that it's clear this is not a final API and it may change at any point without warning - let's add to docblock?  We haven't really designed the user facing side of this yet.

Reply:
sounds good